From 10b7e7dae0ad7ae76ee9259b2d2d2ba70aefed2e Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Thu, 6 Feb 2025 02:07:38 -0700
Subject: [PATCH] Only build CutTools and not its dependencies

* Remove MPFUN90 (MPFUN), OneLoop (AVH), and QCDLoop (QCDLoop) from the ALL
  target defintion and only build CutTools (CTS) as all these dependencies
  are installed from conda-forge.
---
 src/makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/makefile b/src/makefile
index f9f58f0..4e13b65 100644
--- a/src/makefile
+++ b/src/makefile
@@ -88,7 +88,7 @@ qlratreal.o qlratgam.o ddilog.o qllnrat.o qlfunctions.o  qlYcalc.o qlkfn.o \
 qlxpicheck.o
 
 #ALL = $(MPFUN) $(AVH) $(CTS) $(LT) $(QCDLOOP)
-ALL = $(MPFUN) $(AVH) $(CTS) $(QCDLOOP)
+ALL = $(CTS)
 
 libcts.a: libcts.a($(ALL))
 
-- 
2.47.1

